libs/designsystem/src/lib/components/toast/services/toast.helper.ts

```diff
@@ -30,9 +29,18 @@ export class ToastHelper {
 
   private getCssClass(messageType: MessageType): string {
     let cssClass = ToastHelper.CSS_CLASS;
-    if (messageType) {
-      cssClass += ' ' + messageType;
+
+    if (messageType === 'danger') {
+      console.warn(
+        `[DEPRECATED] 'danger' message type is deprecated. Use Alerts for critial warnings. Message type will be set as 'warning'`
+      );
+
+      return (cssClass += ' ' + 'warning');
     }
-    return cssClass;
+    return (cssClass += ' ' + messageType);
```

Comment:
The null check for messageType is gone - why? 🤔

Reply:
This is partly my fault, I suggested this, and dont understand why we need the null check here, when the parameter is typed?

Reply:
Added the null check again